python-avd/pyavd/_eos_designs/structured_config/base/__init__.py

```diff
@@ -406,25 +368,25 @@ def spanning_tree(self) -> None:
             elif spanning_tree_mode == "rstp":
                 self.structured_config.spanning_tree.rstp_priority = priority
 
-    @cached_property
-    def service_unsupported_transceiver(self) -> dict | None:
+    @structured_config_contributor
+    def service_unsupported_transceiver(self) -> None:
         """service_unsupported_transceiver based on unsupported_transceiver data-model."""
-        return self.inputs.unsupported_transceiver._as_dict() or None
+        self.structured_config.service_unsupported_transceiver = self.inputs.unsupported_transceiver
 
-    @cached_property
-    def local_users(self) -> list | None:
+    @structured_config_contributor
+    def local_users(self) -> None:
         """local_users set based on local_users data model."""
         if not self.inputs.local_users:
-            return None
+            return
 
-        return [user._as_dict() for user in self.inputs.local_users._natural_sorted()]
+        for user in self.inputs.local_users._natural_sorted():
+            self.structured_config.local_users.append(user)
```

Comment:
I think this is the correct class already. Try this:
```suggestion
        self.structured_config.local_users = self.inputs.local_users.natural_sorted()
```